rfc/0000-rfc-process.md

```diff
@@ -0,0 +1,60 @@
+---
+name: rfc_process
+started: 2024-07-04
+pr: pact-foundation/roadmap#0000
+---
+## Summary
+
+The Request for Comments (RFC) process is a way to propose and discuss changes to the Pact specification, ecosystem, and community. The RFC process ensures that changes are well-considered, documented, and communicated to all stakeholders. This RFC proposes the process itself, including how to submit an RFC, how to review and discuss it, and how to accept or reject it.
+
+## Motivation
+
+While the evolution of Pact and its ecosystem is open, the process for proposing and discussing changes is not well-defined. By introducing an RFC process, we can ensure that changes are well-considered, documented, and communicated to all stakeholders. This will help to ensure that changes are made in a transparent and collaborative way, and that the community has a voice in the direction of the Pact ecosystem.
+
+## Guide-level explanation
+
+### Scope of RFCs
+
+The Pact ecosystem is broad and includes, broadly speaking, the following areas:
+
+- The Pact specification itself ([pact-foundation/pact-specification](https://github.com/pact-foundation/pact-specification));
+- The Pact broker ([pact-foundation/pact-broker](pact-foundation/pact_broker));
+- The Pact CLI ([pact-foundation/pact-ruby-cl](https://github.com/pact-foundation/pact-ruby-cli)); and,
```

Comment:
link text `pact-foundation/pact-ruby-cl` should be `pact-foundation/pact-ruby-cli`

Reply:
Fixed